website/assets/styles/pages/articles/articles.less

```diff
@@ -50,11 +50,12 @@
       }
       [purpose='article-card-body'] {
         padding: 24px 24px 32px 24px;
-        [purpose='category-link'] {
+        [purpose='category-name'] {
           text-transform: uppercase;
-          color: @core-fleet-black-50;
+          color: @core-vibrant-blue;
```

Comment:
```suggestion
          color: @core-fleet-black-50;
```

Reply:
@eashaw, thanks for spotting that in the wires, but I changed those to fleet black on-the-fly. Also, vibrant-blue should be reserved for links. I keep planning to make time to reify the wireframes. I'll make sure to get that on my to-dos properly.